conf/vhost/git_smart_http.conf

```diff
@@ -1,11 +1,11 @@
-ProxyPass /git !
+ProxyPass _SERVER_PATH_PREFIX_git !
 
 # see https://www.kernel.org/pub/software/scm/git/docs/git-http-backend.html for details
 # needs mod_cgi to work -> a2enmod cgi
 SetEnv GIT_PROJECT_ROOT "_GIT_REPOSITORIES_"
 SetEnv GIT_HTTP_EXPORT_ALL
-ScriptAlias /git/ "_GIT_HTTP_BACKEND_"
-<Location /git>
+ScriptAlias /git_SERVER_PATH_PREFIX_ "_GIT_HTTP_BACKEND_"
```

Comment:
The other way around would be caught by the Alias, see this comment on the OP PR: https://github.com/opf/openproject/pull/3985